### PR TITLE
Adding more instructions to points library readme

### PR DESCRIPTION
**Pull request by mzkrasner**
Adds additional color to the readme to explain intended usage and call out hosted node endpoint for testing only

### CODE REVIEW

File: libraries/points/README.md [hunk 1]
```diff
@@ -1,13 +1,188 @@
...
+// allocate points to track history
+await writer.allocatePointsTo(recipient, amount, {
+    recipient,
+    points: amount,
+    date: new Date().toISOString()
+})
+```
+
+Note: developers who wish to use the allocation and aggregation functionality together must control for keeping the totals for each in sync within their application logic - this is not automatically accounted for in the library.
+
```
Comment (gvelez17) [inline libraries/points/README.md:185]: do you want to link here to pullToPoints as an example of how?  i can do that later in a separate PR.   its currently at https://github.com/ceramicstudio/pullToPoints could be renamed

**Review by gvelez17 — DISMISSED** [review]
LGTM, just one question if we can/should link to a further example since we have one

**Review by PaulLeCam — APPROVED** [review]
Could be worth pointing to the `@ceramic-solutions/points-composite` package as well in case people want to use it directly with ComposeDB?

**Comment by mzkrasner** [comment]
@PaulLeCam composite instructions have been added

**Pull request merged**